goal_src/jak2/engine/ui/bigmap-data.gc

```diff
@@ -7,3 +7,2587 @@
 
 ;; DECOMP BEGINS
 
+(define *bigmap-compressed-layers*
+  (new 'static 'bigmap-compressed-layers
+    :layer0 (new 'static 'bigmap-info-array :data (new 'static 'inline-array bigmap-info 21
```

Comment:
I think that `layer0` isn't a `bigmap-info-array` - these floats don't make any sense.

Looking at the file, it seems like each layer is actually a different size sadly.
So this is probably a `(inline-array uint32)` and we'll need to modify `data_decompile.cpp` (though this is probably an easy one)

Reply:
I agree, was just a guess.  I'll wait for https://github.com/open-goal/jak-project/pull/1957 until fixing this as adding all the special cases for the fields will be a lot easier with my cleanup there.